sample/README.md

```diff
@@ -4,7 +4,7 @@ Sample application for the Urban Airship SDK.
 
 ## Setup
 
-1) Update the [airshipconfig.properties](src/main/assets/airshipconfig.properties) file
+1) Update the [airshipconfig.properties](src/main/assets/airshipconfig.properties.sample) file
```

Comment:
Actually the file needs to be renamed to `airshipconfig.properties`. We named it to sample to avoid so we can gitignore peoples actual config. Could you update the instructions to add a rename/copy step?